doc/radosgw/rgw-cache.rst

```diff
@@ -86,42 +90,62 @@ Put in-place your nginx configuration files and edit them according to your envi
 
-nginx.conf should go to /etc/nginx/nginx.conf  
+`nginx.conf` should go to `/etc/nginx/nginx.conf`  
 
-nginx-lua-file.lua should go to /etc/nginx/nginx-lua-file.lua
+`nginx-lua-file.lua` should go to `/etc/nginx/nginx-lua-file.lua`
 
-nginx-default.conf should go to /etc/nginx/conf.d/nginx-default.conf  
+`nginx-default.conf` should go to `/etc/nginx/conf.d/nginx-default.conf`
 
-The parameters that are most likely to require adjustment according to the environment are located in the file nginx-default.conf   
+The parameters that are most likely to require adjustment according to the environment are located in the file `nginx-default.conf`
 
 Modify the example values of *proxy_cache_path* and *max_size* at:
 
-`proxy_cache_path /data/cache levels=2:2:2 keys_zone=mycache:999m max_size=20G inactive=1d use_temp_path=off;` 
+::
 
-And modify the example *server* values to point to the RGWs URIs: 
+ proxy_cache_path /data/cache levels=2:2:2 keys_zone=mycache:999m max_size=20G inactive=1d use_temp_path=off;
 
-`server rgw1:8000 max_fails=2 fail_timeout=5s;`
 
-`server rgw2:8000 max_fails=2 fail_timeout=5s;`
+And modify the example *server* values to point to the RGWs URIs: 
+
+::
 
-`server rgw3:8000 max_fails=2 fail_timeout=5s;` 
+ server rgw1:8000 max_fails=2 fail_timeout=5s;
+ server rgw2:8000 max_fails=2 fail_timeout=5s;
+ server rgw3:8000 max_fails=2 fail_timeout=5s;
    
-It is important to substitute the access key and secret key located in the nginx.conf with those belong to the user with the amz-cache caps 
+| It is important to substitute the *access key* and *secret key* located in the `nginx.conf` with those belong to the user with the `amz-cache` caps
+| for example, create the `cache` user as following:
+
+::
+
+ radosgw-admin user create --uid=cacheuser --display-name="cache user" --caps="amz-cache=read" --access-key <access> --secret <secret>
 
 It is possible to use nginx slicing which is a better method for streaming purposes.
 
-For using slice you should use nginx-slicing.conf and not nginx-default.conf
+For using slice you should use `nginx-slicing.conf` and not `nginx-default.conf`
 
 Further information about nginx slicing:
 
 https://docs.nginx.com/nginx/admin-guide/content-cache/content-caching/#byte-range-caching
 
 
-If you do not want to use the prefetch caching, It is possible to replace nginx-default.conf with nginx-noprefetch.conf
+If you do not want to use the prefetch caching, It is possible to replace `nginx-default.conf` with `nginx-noprefetch.conf`
 Using noprefetch means that if the client is sending range request of 0-4095 and then 0-4096 Nginx will cache those requests separately, So it will need to fetch those requests twice.
 
 
 Run nginx(openresty):
+
 ::
 
 $ sudo systemctl restart nginx
+
+Appendix
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**A note about performance:** In certain instances like development environment, disabling the authontication by commenting the following line in `nginx-default.conf`:
```

Comment:
sp. authentication

Reply:
thank you, pushed a fix for spelling issues.